json/src/test/java/org/jdbi/v3/json/AbstractJsonMapperTest.java

```diff
@@ -141,6 +141,12 @@ public int getBitcoins() {
 
         @Override
         public boolean equals(Object x) {
+            if (this == x) {
+                return true;
+            } else if (x == null || getClass() != x.getClass()) {
```

Comment:
`x instanceof JsonBean` does both these checks in one expression

Reply:
ok.